experiment/aim.md

```diff
@@ -1 +1,6 @@
-### Aim of the experiment
+Mass spectroscopy the most poweful technique to weigh the molar masses of a given molecule. It is useful in indentifying unkonown compounds and studying their molecular structure. The principle consist of ionizing the sample in the gas phase to formed positively charged ion or radical ions and then measure the mass to charge ratio of all ions. It is largely used in industry and academia and finds application in chemistry, Biotechnology, pharmaceutical, environmental, Clinical, Geological fields. Different methods use different techniques for ionisation which intern determines what types of samples can be analyzed by mass spectrometry. Of all available methods Electron Ionisation or Impact mass spectroscopy is the the simplest of all mass technique that is widely used.  
```

Comment:
1. 'in the gas phase to formed positively charged ion'
Should be 'form' instead of 'formed'

2. 'ionisation which intern determines'
Should be 'ionisation, which in turn determines'